.travis.yml

```diff
@@ -21,3 +21,6 @@ matrix:
 before_install:
   - "export DISPLAY=:99.0"
   - "sh -e /etc/init.d/xvfb start"
+
+env:
+  - DB=postgres
```

Comment:
Since there's a `matrix` above, you should modify the matrix to include 1 of each, something like

```
matrix:
   include:
     - rvm: 2.3.1
       script:
         - bundle exec danger
     - rvm: 2.3.1
        env:
          - DATABASE_ADAPTER=pg
     - rvm: 2.3.1
        env:
          - DATABASE_ADAPTER=mongoid
```

For now feel free to just remove ruby-head and jruby-head.